src/app/menu/menu.component.html

```diff
@@ -14,26 +14,36 @@
       <ul class="navbar-nav ml-auto">
 
         <li class="nav-item">
-          <a href="" class="nav-link">Home</a> <!-- Welcome to football app -->
+          <a href="/home" class="nav-link">Home</a> <!-- Welcome to football app -->
         </li>
         <li class="nav-item dropdown">
-          <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button"
+          <a class="nav-link dropdown-toggle" href="#" id="navDrop1" role="button"
              data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
             Top&nbsp;5&nbsp;Leagues
           </a>
           <div class="dropdown-menu" aria-labelledby="navbarDropdown">
-            <a class="dropdown-item" href="/top5-league" class="nav-link">Premier&nbsp;League</a>
-            <a class="dropdown-item" href="/top5-league" class="nav-link">Bundesliga</a>
-            <a class="dropdown-item" href="/top5-league" class="nav-link">La&nbsp;Liga</a>
-            <a class="dropdown-item" href="/top5-league" class="nav-link">Serie&nbsp;A</a>
-            <a class="dropdown-item" href="/top5-league" class="nav-link">League&nbsp;1</a>
+            <a class="dropdown-item" (click)="navigateToTop5Leagues(2021)">Premier&nbsp;League</a>
+            <a class="dropdown-item" (click)="navigateToTop5Leagues(2002)">Bundesliga</a>
+            <a class="dropdown-item" (click)="navigateToTop5Leagues(2014)">La&nbsp;Liga</a>
+            <a class="dropdown-item" (click)="navigateToTop5Leagues(2019)">Serie&nbsp;A</a>
+            <a class="dropdown-item" (click)="navigateToTop5Leagues(2015)">League&nbsp;1</a>
```

Comment:
Zrób sobie jakąś listę lig z nazwami (trzymaj w serwisie) i tutaj wydrukuj za pomocą ngFor zamiast ręcznie (dodatkowo zamiast navigateToTop5Leagues dałbym navigateToLeague, bo jak kiedyś byś chciał rozszerzyć to nazwa trochę nie pasuje